datafusion/common/src/scalar.rs

```diff
@@ -3300,33 +3409,6 @@ mod tests {
         assert_eq!(result, &expected);
     }
 
-    #[test]
```

Comment:
Is ScalarValue::eq_array test for List elsewhere? Otherwise, we can add the test for it.

Reply:
Sure!